pkg/azurecr/azurecr.go

```diff
@@ -67,28 +67,43 @@ func (d *AzureCR) ListTags(repository string) ([]string, error) {
 	}
 
 	var tagsJSON azureCRTags
-	req, err := http.NewRequest("GET", endpoint, nil)
-	if err != nil {
-		return []string{}, microerror.Mask(err)
+	var tags []string
```

Comment:
When we use `%s/v2/%s/tags/list` - can't we make this whole thing generic? I think this applies to all v2 endpoints.